tfhe/src/c_api/shortint/parameters.rs

```diff
@@ -356,3 +356,15 @@ pub unsafe extern "C" fn shortint_get_parameters_small(
         }
     })
 }
+
+#[repr(C)]
+pub struct CompressionParameters(
+    pub(crate) *const crate::shortint::parameters::glwe_compression::CompressionParameters,
+);
+
+unsafe impl Sync for CompressionParameters {}
```

Comment:
why is it required ?

Reply:
for
```
 #[no_mangle]
pub static COMP_PARAMS_FOR_MESSAGE_2_CARRY_2_KS_PBS
```
which can be accessed from any thread

Reply:
Removed pointer